### ISSUE TITLE
Import varaiables from non .env files stopped working with 6.3.0 

### ISSUE THREAD

**Issue by smaktacular** [AnWeber/vscode-httpyac#184]
Given the following files:

.pat.http
```
@Token = abc
```

debug.http
```
# @import .pat.http

###
GET https://{{url}}/v1.0/debug
Authorization: Bearer {{Token}}
```

Sending the request in debug.http yields 
```
ReferenceError: Token is not defined
	at global.<anonymous> ...
```

In 6.2.0 this works.

**Comment by AnWeber** [comment]
Should be fixed with 6.3.1. please try this version

**Comment by smaktacular** [comment]
nope, still happening with 6.3.1

**Comment by AnWeber** [comment]
Error is fixed with v6.3.2. The cause was an error in the call for executing the imported global regions. The current file was used instead of the imported one. Test is created for it.
